src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+
+    public Figure getRandomFigure() {
```

Comment:
Replace your logic here: FigureSupplier with public Figure getRandomFigure() method - for generating figure with random properties.

Reply:
Could you please explain clearlier